README.md

```diff
@@ -4,4 +4,36 @@ This is the code behind https://crowdstrike.github.io/community/
 On that webpage, you can find content such as:
 - [List of CrowdStrike's Open Source Projects](https://crowdstrike.github.io/community/)
 - [Guidance for CrowdStrike Engineering](https://crowdstrike.github.io/community/open-source-policy/#engineering)
-- [Guidance for All Other Departments/Orgs](https://crowdstrike.github.io/community/open-source-policy/#everyone-else)
\ No newline at end of file
+- [Guidance for All Other Departments/Orgs](https://crowdstrike.github.io/community/open-source-policy/#everyone-else)
+
+
+
+## Developing the Site
+
+1. Ensure ``bundler`` and `jekyll` and ``rake`` are installed locally:
+```shell
+gem install bundler jekyll rake
+```
+2. Install NPM dependencies. The webroot is the `docs` folder, so you must change into that directory:
+```shell
+cd docs/ && npm install
+```
```

Comment:
```suggestion
2. Change directory to the `docs` folder and use `bundler` to build the site:
```shell
cd docs/ && bundle exec jekyll build
```
```

Reply:
If you don't `npm install` then the site will fail to build (many missing modules)

Reply:
Odd.... I haven't seen that behavior at all. I don't even have npm installed. Is this a macos requirement?

Reply:
Dunno. I do develop on OSX, and if the node modules are missing or outdated, a bunch of errors are thrown and the site doesn't build locally.

Reply:
funny... okay. I'll merge and we can fix later.

Reply:
( I can't duplicate anymore, since, well, they're all installed now)